internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -44,15 +54,38 @@ func getBillingModelForCloudProvider(cloudProvider string) (amsv1.BillingModel,
 	return "", errors.InvalidBillingAccount("unsupported cloud provider")
 }
 
-func newBaseQuotaReservedResourceResourceBuilder() amsv1.ReservedResourceBuilder {
-	rr := amsv1.ReservedResourceBuilder{}
-	rr.ResourceType("cluster.aws") //cluster.aws
-	rr.BYOC(false)                 //false
-	rr.ResourceName("rhosak")      //"rhosak"
-	rr.BillingModel("marketplace") // "marketplace" or "standard"
-	rr.AvailabilityZoneType("single")
```

Comment:
We were always setting the `AvailabilityZoneType` value to "single". In this PR we also updated the logic to set it to "single" or "multi" depending on the MultiAZ value of the kafka instance

Reply:
Qn: 

Will there be a need to perform a migration to `multi` for already existing values?

Reply:
It will be inaccurate for the already existing clusters but there is no need as we were told it is merely descriptive

Reply:
Thanks.